delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ProjectsRoutes.scala

```diff
@@ -65,30 +65,26 @@ final class ProjectsRoutes(identities: Identities, acls: Acls, projects: Project
   private def fetchByUUID(orgUuid: UUID, projectUuid: UUID, permission: Permission)(implicit
       caller: Caller
   ): Directive1[ProjectResource] =
-    onSuccess(projects.fetch(projectUuid).runToFuture).flatMap {
-      case Some(project) =>
-        authorizeFor(AclAddress.Project(project.value.ref), permission).tflatMap { _ =>
-          if (project.value.organizationUuid == orgUuid)
-            provide(project)
-          else
-            Directive(_ => discardEntityAndEmit(ProjectNotFound(orgUuid, projectUuid): ProjectRejection))
+    onSuccess(projects.fetch(projectUuid).attempt.runToFuture).flatMap {
```

Comment:
Yes, but it won't work because of this weird logic of returning `AuthorizationFailed` instead of `NotFound`. If we use
`projects.fetch(orgUuid, projectUuid)` it will return `ProjectNotFound` if the `projectUuid` is not found and also if `orgUuid` does not match the found project by `projectUuid`. This is the correct behaviour, but notice that here we do this extra check on `authorizeFor`.